### ISSUE TITLE
pressDrag event problem!

### ISSUE THREAD

**Issue by thanhluantl2304** [tongyy/react-native-draggable#31]
I had a draggable and have set the pressDrag event. It was fine when i debugged in simulator. But when i debugged in my device. It's hard to press the button. The pressDragRelease worked fine but the pressDrag. Can you help me about that issue?

**Comment by tongyy** [comment]
Hi there, @thanhluantl2304 
Is your device an android phone? You can use pressOutDrag instead. These event are triggered  from pressInDrag->pressOutDrag->pressDrag. Can you test pressOutDrag and give me a quick reply?
I think it's related to [issue20](https://github.com/tongyy/react-native-draggable/issues/20)

**Comment by thanhluantl2304** [comment]
Hi tongyy, thanks for quick reply!. I try pressOutDrag and longPressDrag also. It works fine but when i want to drag the button. These events fire too. That's quite confusing! In my app, the draggable button is to navigate to other screen. But i faced that problem. My friend's android device works fine but my device. My device use android 9. Thanks

**Comment by tongyy** [comment]
https://github.com/facebook/react-native/issues/22397
It seems onPress event of TouchableOpacity is still buggy on Android phones.
There is an idea. When you are dragging it, the events are triggered like this orders pressInDrag-> longPressDrag->pressOutDrag. You can pass a function as longPressDrag to prevent calling pressOutDrag. 

```
longPressDragHanlder() {
  needsToRun = false;
}
pressOutDragHandler() {
  if(needsToRun) {
   run();
  needsToRun = true;
  }
  
}
```

**Comment by thanhluantl2304** [comment]
Thank you! But i consider this resolution. [https://stackoverflow.com/questions/47568850/touchableopacity-with-parent-panresponder].
I've changed in your code and comment line "onMoveShouldSetPanResponderCapture" then it worked. Regards!

**Comment by thanhluantl2304** [comment]
Sorry for late reply! How can i contribute my code? I've just tested on Android and it worked perfect! I will test it on IOS later and will reply you soon! Thank you!

**Comment by tongyy** [comment]
I don't have many strict rules as long as you just run the command which is npm run lint before you commit. Since I don't have my computers to maintain this repo recently, I appreciate your contribution and welcome to join us :) 
I usually do as follows:
1. fork this project into your repo.
2. commit and push into your branch
3. open the pull request 
4. I will review it and then merge it if there is no problem

**Comment by thanhluantl2304** [comment]
yeah, i will try it when i have time! Thank you again!

**Comment by tongyy** [comment]
it has fixed in Version 2.0.0
